core/app/models/concerns/spree/active_storage_adapter/attachment.rb

```diff
@@ -59,12 +61,25 @@ def metadata
         @attachment.metadata
       end
 
-      def normalize_styles(styles)
-        styles.transform_values { |v| v.split('x').map(&:to_i) }
+      def styles_to_transformations(styles)
+        styles.transform_values(&method(:imagemagick_to_image_processing_definition))
+      end
+
+      def imagemagick_to_image_processing_definition(definition)
+        width_height = definition.split('x').map(&:to_i)
+
+        case definition[-1].to_sym
+        when :>
+          { resize_to_limit: width_height }
+        when :^
+          { resize_to_fill: width_height }
+        else
+          { resize_to_fit: width_height }
```

Comment:
This is the case for `:^`, BUT this upscales the image, if the original dimensions are smaller than the requested dimensions. Not sure if we want that as default? We probably want `resize_to_limit` as default.

Reply:
>  We probably want resize_to_limit as default.

Thanks, I think you're right, plus it won't change the previous behavior. Fixed.